dashboard/app/models/game.rb

```diff
@@ -141,6 +141,10 @@ def supports_sharing?
     app == TURTLE || app == FLAPPY || app == BOUNCE || app == STUDIO || app == STUDIO_EC || app == APPLAB || app == CRAFT || app == GAMELAB
   end
 
+  def sharing_filtered?
```

Comment:
That is correct. This is just currently the case for STUDIO (blockly mode) apps because they are the only way of sharing free-text programs in CSF.